src/bors/handlers/mod.rs

```diff
@@ -28,10 +30,26 @@ pub async fn handle_bors_event<Client: RepositoryClient>(
     event: BorsEvent,
     state: Arc<dyn BorsState<Client>>,
     ctx: Arc<BorsContext>,
+) -> anyhow::Result<()> {
+    match event {
+        BorsEvent::Repository(event) => {
+            handle_bors_repository_event(event, state, ctx).await?;
+        }
+        BorsEvent::Global(event) => {
+            handle_bors_global_event(event, state, ctx).await?;
+        }
+    }
+    Ok(())
+}
+
+pub async fn handle_bors_repository_event<Client: RepositoryClient>(
+    event: BorsRepositoryEvent,
+    state: Arc<dyn BorsState<Client>>,
+    ctx: Arc<BorsContext>,
 ) -> anyhow::Result<()> {
     let db = Arc::clone(&ctx.db);
     match event {
-        BorsEvent::Comment(comment) => {
+        BorsRepositoryEvent::Comment(comment) => {
             // We want to ignore comments made by this bot
             if let Some(repo) = get_repo_state(state, &comment.repository) {
```

Comment:
It would be nice to attach the repository name to each `BorsRepositoryEvent`, so that we can move out `get_repo_state` out of the match